.github/workflows/operate-ci.yml

```diff
@@ -72,7 +72,7 @@ jobs:
       branch: ${{ github.head_ref || github.ref_name }}
       command: mvn -f operate/qa/integration-tests verify -P operateItStage1 -B -DskipChecks -Dfailsafe.rerunFailingTestsCount=2
       test-type: Integration Tests
-      runner-type: gcp-core-32-default
+      runner-type: gcp-core-32-longrunning
```

Comment:
❌ Revert this before merge

Motivation: this increases costs significantly since the workflow is run very often + according to [Slack](https://camunda.slack.com/archives/C071KP5BTHB/p1726475962618189?thread_ts=1726222151.891509&cid=C071KP5BTHB) the root cause has been found to be not Infra problems (which makes sense since the problematic behavior was only noticed on this PR which usually indicates that the problem lies in the PR itself).